<?xml version="1.0"?>
<launch>
	<param name="/maxSteer" value="27.2"/>
	<param name="/mission_type" value="staticA"/>

	<node name = "staticA" pkg = "mission_commander" type = "mission_commander_node.py" output="screen"/>

</launch>
